readme.md

```diff
@@ -91,6 +91,7 @@ class Example extends Component {
  errorColor            | Text field color for errored state          |   String | rgb(213, 0, 0)
  disabledLineType      | Text field line type in disabled state      |   String | dotted
  animationDuration     | Text field animation duration in ms         |   Number | 225
+ helperNumberOfLines   | Text field helper number or lines           |   Number | -
```

Comment:
It would be nice if you didn't have to know this in advance, but for the component to just handle text of any length.

Reply:
Handle text of any length requires to calculate the length based on styling and container size. This is made via callback to native side which doesn’t match the component lifecycle.

Reply:
Knowing the number of lines you require is just not realistic though. On an iphone X it may be one line, but an iphone 4 it may be 3.